firestore-genai-chatbot/functions/src/config.ts

```diff
@@ -102,10 +81,10 @@ function getSafetySettings(): GoogleAISafetySetting[] | VertexSafetySetting[] {
 
 const config: Config = {
   vertex: {
-    model: getModel(),
+    model: process.env.MODEL!,
```

Comment:
Should we be handling invalid model name in any way? I see that we don't have any try/catch blocks or logs for that and `generative-ai` doesn't seem to validate it in any way until the HTTPS [request is sent](https://github.com/google/generative-ai-js/blob/main/packages/main/src/methods/generate-content.ts#L55-L60).

Reply:
Yeah been told they prefer not to validate it:

> I think skipping the validation would be worth having to update the extension every single time Google decides to update a model.

Reply:
Gotcha! Also fair point 😂